docs/src/pages/components/autocomplete/autocomplete.md

```diff
@@ -116,10 +116,16 @@ It accepts almost the same options as the Autocomplete component minus all the p
 related to the rendering of JSX.
 The Autocomplete component is built on this hook.
 
-```jsx
+```js
```

Comment:
**Off-topic**

I wonder if we shouldn't always use tsx so we don't have to think to much about it.

Reply:
Yeah, why not